astropy/coordinates/tests/test_polarization.py

```diff
@@ -36,7 +36,11 @@ def test_vector_list_init():
 
 def test_undefined():
     sk = StokesCoord(np.arange(-10, 7))
-    assert_equal(sk.symbol, "? ? YX XY YY XX LR RL LL RR ? I Q U V ? ?".split())
+    assert_equal(
+        sk.symbol,
+        ["?", "?", "YX", "XY", "YY", "XX", "LR", "RL"]
+        + ["LL", "RR", "?", "I", "Q", "U", "V", "?", "?"],
+    )
```

Comment:
Why is this better than a `noqa`?

Reply:
My impression is that in astropy's culture, noqas are a last resort and should be avoided as much as possible. Ultimately I don't have a strong opinion.

Reply:
A `noqa` is a code smell, but here it seems justified.

Reply:
Some future version of Ruff could well have a rule about preforming runtime addition of list literals with an automatic fix that would write out a single list literal directly. By keeping the code as is and adding a `noqa` we can be much more confident that future Ruff updates will not force us to revisit this line.